webapp/src/templates/partials/contacts_content.html

```diff
@@ -105,7 +105,7 @@ <h3 translate>clinic.field.children</h3>
                 heading="child.doc.name"
                 dob="child.doc.date_of_birth"
                 primary-contact="child.isPrimaryContact"
-                task-count="child.taskCount"
+                task-count="$ctrl.selected.tasksByContact && $ctrl.selected.tasksByContact[child.doc._id]"
```

Comment:
I don't think you need to do the undefined check - angularjs should handle this gracefully and just pass the undefined.

Reply:
You're right! Thanks!